cost/aws/unused_clbs/aws_unused_clbs.pt

```diff
@@ -865,6 +868,13 @@ policy "pol_unused_clbs" do
       field "service" do
         label "Service"
       end
+      field "resourceARN" do
+        label "Resource ARN"
+      end
+      field "resourceName" do
```

Comment:
@XOmniverse is this `resourceName` field redundant as is the same as `resourceID`?